homework-8/script-1.js

```diff
@@ -0,0 +1,142 @@
+/*Створити масив, довжину та елементи якого задає користувач (через prompt).
+ Елементами масиву повинні бути числа*/
+const arrayNumber = Array();
+const lengthArray = +prompt("Enter please the length of array");
+if (!Number.isInteger(lengthArray)) {
+    console.log("Sorry, we cannot to process data");
+} else {
+    for (let i = 0; i < lengthArray; i++) {
+        let elementArray = +prompt("Enter please number");
+        if (!Number.isInteger(elementArray)) {
+            console.log("Sorry, elements of array need to be numbers");
+        } else {
+            arrayNumber.push(+elementArray);
```

Comment:
You are already doing +prompt. No need it here

```suggestion
            arrayNumber.push(elementArray);
```